hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/kafka/KafkaConfiguration.java

```diff
@@ -115,6 +129,22 @@ MultiDCAwareService multiDCAwareService(KafkaNamesMappers kafkaNamesMappers, Sch
                 ofSeconds(subscriptionProperties.getOffsetsMovedTimeoutInSeconds()));
     }
 
+    private SubscriptionOffsetChangeIndicator getRepository(
+            List<DcBoundRepositoryHolder<SubscriptionOffsetChangeIndicator>> repostories,
+            KafkaProperties kafkaProperties) {
+        return repostories.stream()
+                .filter(x -> kafkaProperties.getDc().equals(x.getDcName()))
+                .findFirst().orElseThrow(() ->
+                        new IllegalArgumentException(
+                                String.format("Kafka brokers cluster name %s not matched with Zookeeper cluster name %s",
```

Comment:
It would be nice to update this message since you have changed filter predicate. Maybe something like this will be fine: `"Kafka cluster dc name '%s'  not matched with Zookeeper dc names `%s`", kafkaProperties.getDc(), ...`